<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PVP_Client</class>
 <widget class="QWidget" name="PVP_Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1081</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PVP_Client</string>
  </property>
  <widget class="QWidget" name="Main_UI" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <widget class="QPushButton" name="back_Button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>78</y>
      <width>90</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Resource/UI_Design/#Block.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../../Resource/Gobang.qrc">
      <normaloff>:/Resource/UI_Design/Fun4_BackButton.png</normaloff>:/Resource/UI_Design/Fun4_BackButton.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>90</width>
      <height>50</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="playerInf_1">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>252</x>
      <y>616</y>
      <width>159</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;border-width:0;border-style:outset;
color:black</string>
    </property>
    <property name="text">
     <string>玩家一 (黑)</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_1">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>872</y>
      <width>275</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;border-width:0;border-style:outset;
color:&quot;grey&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>玩家一</string>
    </property>
   </widget>
   <widget class="QLabel" name="Inf_Card_1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>580</y>
      <width>353</width>
      <height>205</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Resource2/UI_Design/Inf_Card.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="character_1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>838</y>
      <width>105</width>
      <height>105</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Resource2/UI_Design/character_7.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="pieceInf_1">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>252</x>
      <y>708</y>
      <width>159</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;border-width:0;border-style:outset;
color:red</string>
    </property>
    <property name="text">
     <string>轮到我方落子</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLCDNumber" name="time_1">
    <property name="geometry">
     <rect>
      <x>254</x>
      <y>663</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Outline</enum>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1387</x>
      <y>131</y>
      <width>275</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Taipei Sans TC Beta</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;border-width:0;border-style:outset;color:grey</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>玩家二</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pieceInf_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1630</x>
      <y>390</y>
      <width>159</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;border-width:0;border-style:outset;
color:black</string>
    </property>
    <property name="text">
     <string>等待我方落子</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="playerInf_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1630</x>
      <y>299</y>
      <width>159</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;border-width:0;border-style:outset;color:black</string>
    </property>
    <property name="text">
     <string>玩家二 (白)</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="character_2">
    <property name="geometry">
     <rect>
      <x>1680</x>
      <y>94</y>
      <width>105</width>
      <height>105</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Resource2/UI_Design/character_6.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLCDNumber" name="time_2">
    <property name="geometry">
     <rect>
      <x>1630</x>
      <y>347</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Outline</enum>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="Inf_Card_2">
    <property name="geometry">
     <rect>
      <x>1496</x>
      <y>263</y>
      <width>353</width>
      <height>205</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Resource2/UI_Design/Inf_Card.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="Trace_Inf">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1643</x>
      <y>926</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;color:black</string>
    </property>
    <property name="text">
     <string>本模式仅一次机会</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <zorder>Inf_Card_2</zorder>
   <zorder>back_Button</zorder>
   <zorder>name_1</zorder>
   <zorder>Inf_Card_1</zorder>
   <zorder>pieceInf_1</zorder>
   <zorder>time_1</zorder>
   <zorder>character_1</zorder>
   <zorder>playerInf_1</zorder>
   <zorder>name_2</zorder>
   <zorder>playerInf_2</zorder>
   <zorder>character_2</zorder>
   <zorder>time_2</zorder>
   <zorder>pieceInf_2</zorder>
   <zorder>Trace_Inf</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../Resource/Gobang.qrc"/>
  <include location="../../../Resource/Gobang2.qrc"/>
 </resources>
 <connections/>
</ui>
